src/config/providers.ts

```diff
@@ -0,0 +1,13 @@
+// import Portis from '@portis/web3';
+import { InjectedConnector } from '@web3-react/injected-connector';
+
+export const injectedProvider = new InjectedConnector({
+  supportedChainIds: [
+    // temporary
```

Comment:
Even thou it's temporary, I'd save [30,31] as a constant to avoid confusing bugs in the future

Reply:
Ok will create a constant with that. We will propably build it from your `ChainEnum`  when it's ready